app/src/main/AndroidManifest.xml

```diff
@@ -18,13 +18,14 @@
     package="com.example.android.dessertclicker">
 
     <application
-        android:allowBackup="true"
+        android:allowBackup="false"
```

Comment:
Let's actually leave this as true and ignore the warning because all other sample apps have this set to true

Reply:
Noted, I'll change back to true.